tests/integration/models/test_restart.py

```diff
@@ -0,0 +1,151 @@
+"""Test for restart of a model.
+
+Here, exemplarily for a mixed-dimensional poromechanics model with
+time-varying boundary conditions.
+
+"""
+from __future__ import annotations
+
+import json
+import os
+import shutil
+from pathlib import Path
+
+import numpy as np
+import pytest
+
+import porepy as pp
+
+from ...unit.test_vtk import _compare_pvd_files, _compare_vtu_files
+from .test_poromechanics import TailoredPoromechanics
+
+# Store current directory
+current_dir = os.path.dirname(os.path.realpath(__file__))
+
+
+def create_fractured_setup(
+    solid_vals: dict, fluid_vals: dict, uy_north: float, restart: bool
+):
+    """Create a setup for a fractured domain.
+
+    This is an enhanced copy of .test_poromechanics.create_fractured_setup.
+    It enables multiple time steps, and the export of the solution.
+
```

Comment:
Is there a way in which you can import and enhance, rather than copy? The latter more than doubles the maintenance workload and increases the chance of errors.